src/config/__tests__/config.test.ts

```diff
@@ -1,73 +1,19 @@
-import * as path from 'path';
```

Comment:
It feels a little odd having a config.test.ts when we dont have any sort of config.ts or config directory or anything that actually exists called config. I guess its more of a concept in the code, but its not a concept to the users because all they know about are rulesets? 

Should we add "eradicate config" to the tech debt story?

Reply:
Yeah, it's mostly an issue with proper naming. I did update both the code and tests to cover the recent ruleset change, but the naming itself wasn't updated as at the time I worked on that code I thought it'd be called `config` rather than `ruleset`